packages/contracts/src/plugin/WhiteListPlugin.sol

```diff
@@ -18,27 +18,27 @@ abstract contract WhiteListPlugin {
     }
 
     /// @notice - Address of the White List Provider.
-    IWhiteListProvider public whiteListProvider;
+    IWhiteListProvider public immutable WHITELIST_PROVIDER;
```

Comment:
All uppercase is for [constants](https://docs.soliditylang.org/en/v0.8.26/style-guide.html#constants) not `immutable` state variables (which are just state variables).